src/Notepads/Controls/TextEditor/TextEditorCore.cs

```diff
@@ -267,15 +267,28 @@ public void SetScrollViewerPosition(double horizontalOffset, double verticalOffs
         //TODO This method I wrote is pathetic, need to find a way to implement it in a better way 
         public void GetCurrentLineColumn(out int lineIndex, out int columnIndex, out int selectedCount)
         {
-            if (_isLineCachePendingUpdate)
+            /*if (_isLineCachePendingUpdate)
             {
                 _contentLinesCache = (_content + RichEditBoxDefaultLineEnding).Split(RichEditBoxDefaultLineEnding);
                 _isLineCachePendingUpdate = false;
-            }
+            }*/
 
             GetTextSelectionPosition(out var start, out var end);
 
-            lineIndex = 1;
+            lineIndex = (_content + RichEditBoxDefaultLineEnding).Substring(0, start).Length 
```

Comment:
I still think the older way is better. Memory is not a concern here since Notepads cannot handle large file at first place and the string replace here is an overhead when there is no text change during user changing selection and cursor position.

Reply:
You can keep the old method but it needs some changes to match selected items with VS Code.

Reply:
Can you keep the original method as it is and put your new logic in a new method "public void GetCurrentLineColumn2(out int lineIndex, out int columnIndex, out int selectedCount)" and let's decide later since there are already too many changes with in this release. I wan to keep everything minimum. We can then revisit it later.

Reply:
Made changes👍